_includes/ome2021_program_agenda.html

```diff
@@ -19,7 +19,8 @@ <h4>Schedule</h4>
 <p>
     The content will repeated over two sessions to accomodate our
     community across timezones.<br/>
-    All times are in UK - British Summer Time (GMT+1).
+    All times are in UK - British Summer Time (GMT+1). Click on the
```

Comment:
```suggestion
    All times are in UK - British Summer Time (UTC+1). Click on the
```

Proposing to unify with the time format below and remove all mentions of GMT

Reply:
:+1: